authorizer-app-backend/build.gradle.kts

```diff
@@ -11,6 +12,10 @@ plugins {
 
 application {
     mainClass.set("org.radarbase.authorizer.Main")
+    applicationDefaultJvmArgs = listOf(
+        "-Djava.security.egd=file:/dev/./urandom",
```

Comment:
Why do we use this? Just for my interest.

Reply:
It tells Java that it shouldn't rely on the entropy of the machine (on VM's there is typically too little entropy), but use a pseudorandom generator. This avoids any delays in SSL processing where randomness is required -- see e.g. https://www.baeldung.com/java-security-egd#10-what-about-virtual-machines